core/modules/layout/css/layout.flexible.admin.css

```diff
@@ -24,13 +24,13 @@
 }
 .layout--flexible .layout-editor-block-title {
   padding: 0.4em 1em;
-  margin-bottom: 10px;
+  margin:  0 -.9375rem .9375rem;
```

Comment:
Margins on block titles on the manage blocks page are done in `em`s. Can we do that here too?

```
margin: 0 0.5em 0 0; /* LTR */
```

Reply:
Or... is this to counter-act the bootstrap grid? If so we also need to do the same for all other screen sizes.... That might be a lot of work.

Reply:
okay, it does look like we are trying to counteract the bootstrap grid:
```
.container-fluid {
    padding-right: .9375rem;
    padding-left: .9375rem;
}
```
Fortunately this padding is the same at all breakpoints :) p
lease ignore this whole thread, I'll close it.

Reply:
Thanks.